org.eclipse.jdt.ui/ui/org/eclipse/jdt/ui/wizards/NewJavaProjectWizardPageOne.java

```diff
@@ -1831,4 +1831,9 @@ private static boolean isValidWorkingSet(IWorkingSet workingSet) {
 
 		return true;
 	}
+
+	@Override
+	public  org.eclipse.swt.graphics.Point getMinimumPageSize() {
+		 return new org.eclipse.swt.graphics.Point(635,845);
```

Comment:
Such hardcoding makes the change impossible to change as minimum sizes are calculated based on native widgets and that is affected even by changing a UI theme. I strongly believe this is wrong workaround and it will bite us back soon enough if applied.